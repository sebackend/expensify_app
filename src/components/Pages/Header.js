import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <nav>
      <ul>
        <li>
          <NavLink to="/" activeClassName="selected" exact={true} >Dashboard</NavLink>
        </li>
        <li>
          <NavLink to="/create" activeClassName="selected" >Create Expense</NavLink>
        </li>
        <li>
          <NavLink to="/help" activeClassName="selected" >Help</NavLink>
        </li>
      </ul>
    </nav>
  </header>
);

export default Header;