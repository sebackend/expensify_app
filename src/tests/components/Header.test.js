// react-test-renderer
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Pages/Header';

test('should render Header correctly', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});