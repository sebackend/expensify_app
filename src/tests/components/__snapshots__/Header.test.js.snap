// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <nav>
    <ul>
      <li>
        <NavLink
          activeClassName="selected"
          exact={true}
          to="/"
        >
          Dashboard
        </NavLink>
      </li>
      <li>
        <NavLink
          activeClassName="selected"
          to="/create"
        >
          Create Expense
        </NavLink>
      </li>
      <li>
        <NavLink
          activeClassName="selected"
          to="/help"
        >
          Help
        </NavLink>
      </li>
    </ul>
  </nav>
</header>
`;
